charts/fullstack-deployment/templates/network-node-statefulset.yaml

```
@@ -341,6 +346,68 @@ spec:
+        securityContext:
+          {{- include "fullstack.hedera.security.context" . | nindent 10 }}
+        command:
+          - /usr/bin/env
+          - python3.7
+          - /usr/local/bin/mirror.py
+          - --linux
+          - --watch-directory
+          - /opt/hgcapp/recordStreams/sidecar
+          - --s3-endpoint
+          - http://{{ $minioserver.tenant.name }}-hl:9000
+        volumeMounts:
+          - name: hgcapp-record-streams-sidecar
+            mountPath: /opt/hgcapp/recordStreams/sidecar
+            subPath: record{{ $node.accountId }}
+        env:
+          - name: DEBUG
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.debug ($recordStreamSidecar.config).debug | quote }}
+          - name: REAPER_ENABLE
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.reaper.enabled (($recordStreamSidecar.config).reaper).enabled | quote }}
+          - name: REAPER_MIN_KEEP
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.reaper.minKeep (($recordStreamSidecar.config).reaper).minKeep | quote }}
+          - name: REAPER_INTERVAL
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.reaper.interval (($recordStreamSidecar.config).reaper).interval | quote }}
+          - name: REAPER_DEFAULT_BACKOFF
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.reaper.defaultBackoff (($recordStreamSidecar.config).reaper).defaultBackoff | quote }}
+          - name: STREAM_FILE_EXTENSION
+            value: "rcd"
+          - name: STREAM_SIG_EXTENSION
+            value: "rcd_sig"
+          - name: STREAM_EXTENSION
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.compression ($recordStreamSidecar.config).compression | ternary "rcd.gz" "rcd" | quote }}
+          - name: SIG_EXTENSION
+            value: "rcd_sig"
+          - name: RECORD_STREAM_COMPRESSION
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.compression ($recordStreamSidecar.config).compression | quote }}
+          - name: RECORD_STREAM_SIDECAR
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.sidecar ($recordStreamSidecar.config).sidecar | quote }}
+          - name: SIG_REQUIRE
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.signature.require (($recordStreamSidecar.config).signature).require | quote }}
+          - name: SIG_PRIORITIZE
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.signature.prioritize (($recordStreamSidecar.config).signature).prioritize | quote }}
+          - name: BUCKET_PATH
+            value: "recordstreams/record{{ $node.accountId }}"
+          - name: BUCKET_NAME
+            value: {{ $cloud.buckets.streamBucket | quote }}
```

```suggestion
          - name: BUCKET_PATH
            value: "recordstreams/record{{ $node.accountId }}/sidecar"
          - name: BUCKET_NAME
            value: {{ $cloud.buckets.streamBucket | quote }}
```
Based on Ivo's comment: https://swirldslabs.slack.com/archives/C06FWU24B7Y/p1720445868302879

---

Fixed